.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
```

Comment:
I'd recommend dropping `assignees` and `labels`.  That's more our job.

Reply:
These fields are not visible when creating an issue, only relevant for API calls. Omitting them here is the same thing as setting to empty values.